<launch>
  <rosparam command="load" file="$(find simple_bsn)/config/condition.yaml"/>
  <node name= "bsn_handler"  pkg="simple_bsn" type="bsn_handler.py" output="screen">
  </node>
</launch>
